src/Polly/Polly.csproj

```diff
@@ -15,7 +15,6 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Authors>Michael Wolfenden, App vNext</Authors>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
-    <LangVersion>latest</LangVersion>
```

Comment:
Did you mean to remove this?

Reply:
yeah. it is already in the dir props. i can do a new PR if u want

Reply:
No, that's fine. Fewer PRs is less to review 😄